app/Models/Action.php

```diff
@@ -36,7 +36,7 @@ class Action extends Model
      *
      * @var array
      */
-    public static $indexes = ['id', 'callpower_campaign_id'];
+    public static $indexes = ['id', 'campaign_id', 'callpower_campaign_id'];
```

Comment:
Is this already an index in the database?

Reply:
It is!